### PR TITLE
Feature/implementation feature files enrique

### PR DESCRIPTION
**Pull request by Enrique-C**

### CODE REVIEW

**Review by Amet015 — APPROVED** [review]

**Review by Amet015 — APPROVED** [review]

File: trello/api/features/negative/ListNegative.feature [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Created by Enrique Carrizales at 2/10/2020
+Feature: List
+  As a regular user, It manages a List.
+
+  Scenario: A list without name could not be crated
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/negative/ListNegative.feature]: **created**

File: trello/api/features/functional/ListFunctional.feature [hunk 2]
```diff
@@ -17,11 +17,85 @@ Feature: List
...
+    And Validates schema with "list_schema.json"
+    And Sets a "GET" request to "/lists/SecondObjectList.id"
+    And Sends request
+    And Should return status code 200
+    And Sets a "DELETE" request to "/board/BoardObject.id"
+    And Sends request
+    And Should return status code 200
+
+
+  Scenario: Modify the list position to "bottom"
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/functional/ListFunctional.feature]: delete double quotes please **bottom**

File: trello/api/features/negative/OrganizationNegative.feature [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Created by Enrique Carrizales at 2/10/2020
+Feature: Organization
+  As a regular user, It manages a Organization.
+
+  @defect
+  Scenario: API can't create a Organization without name
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/negative/OrganizationNegative.feature]: improve this title, for exaple to Organization can't be created without name

File: trello/api/features/negative/OrganizationNegative.feature [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    And Sends request
+    Then Should return status code 400
+    And Validates response message with message "Display Name must be at least 1 character"
+    And Sets a "GET" request to "/organizations/organizationObject.id"
+    And Sends request
+    And Should return status code 404
+
+
+  @defect
+  Scenario: API can't create a Organization with a wrong website
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/negative/OrganizationNegative.feature]: improve this title please, for example to Organization ca't be created with a wrong website addrees

File: trello/api/features/functional/ListFunctional.feature [hunk 5]
```diff
@@ -2,6 +2,35 @@
...
+    And Saves response as "BoardObject"
+    When Sets a "POST" request to "/lists/"
+      | key     | value            |
+      | name    | functionalList   |
+      | idBoard | (BoardObject.id) |
+      | pos     | top              |
+    And Sends request
+    Then Should return status code 200
+    And Saves response as "ListObject"
+        And Validates response body with
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/functional/ListFunctional.feature]: check identation

File: trello/api/features/functional/ListFunctional.feature [hunk 6]
```diff
@@ -44,6 +68,76 @@ Feature: List
     And Should return status code 200
 
 
+  Scenario: Create a list from another list
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/functional/ListFunctional.feature]: **Creates**

File: trello/api/features/functional/ListFunctional.feature [hunk 7]
```diff
@@ -44,6 +68,76 @@ Feature: List
...
+    And Validates schema with "list_schema.json"
+    And Sets a "GET" request to "/lists/SecondObjectList.id"
+    And Sends request
+    And Should return status code 200
+    And Sets a "DELETE" request to "/board/BoardObject.id"
+    And Sends request
+    And Should return status code 200
+
+
+  Scenario: Modify the list position to bottom
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/functional/ListFunctional.feature]: **Modifies**

File: trello/api/features/negative/ListNegative.feature [hunk 8]
```diff
@@ -0,0 +1,36 @@
+# Created by Enrique Carrizales at 2/10/2020
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/negative/ListNegative.feature]: delete this line

File: trello/api/features/negative/OrganizationNegative.feature [hunk 9]
```diff
@@ -0,0 +1,29 @@
+# Created by Enrique Carrizales at 2/10/2020
```
Comment (JuanCarlosMartinez4) [inline trello/api/features/negative/OrganizationNegative.feature]: delete this line of code

**Review by JuanCarlosMartinez4 — APPROVED** [review]

**Pull request merged**